regions/securityhub.json

```diff
@@ -0,0 +1,24 @@
+[
+  "ap-northeast-1",
+  "ap-south-1",
+  "ap-southeast-1",
+  "ap-southeast-2",
+  "eu-central-1",
+  "eu-south-1",
+  "me-south-1",
+  "us-east-1",
+  "us-east-2",
+  "us-west-1",
+  "af-south-1",
+  "ap-northeast-2",
+  "ca-central-1",
+  "eu-north-1",
+  "eu-west-2",
+  "eu-west-3",
+  "sa-east-1",
+  "us-gov-east-1",
+  "us-gov-west-1",
+  "us-west-2",
+  "ap-east-1",
+  "eu-west-1"
+]
```

Comment:
These region lists are in different orders, which might make keeping them in sync harder… Also, would it make sense to only enable Guard Duty/Security Hub in regions we use, and just block the use of other regions?

Reply:
Oh, wait, it looks like these are generated by `regions.tf`, right? In which case, do they need to be committed? Or have a comment saying they're generated?

Reply:
The [MoJ Security Guidance](https://ministryofjustice.github.io/security-guidance/baseline-aws-accounts/#securityhub) specifies that GuardDuty and SecurityHub should be enabled in all regions, at all times, hence this fetches the regions that the service is available in, which then generates custom providers for each region so this module can enable the service in each one (if the region is also enabled in the account this in run on).

I'll add a comment in the README, because JSON doesn't support comments 👍🏼

Reply:
After having a look at this, we can remove the `regions/*.json` files from being committed, which will be in the next PR.